src/App.vue

```diff
@@ -14,6 +23,11 @@ export default {
   components: {
     NavigationDrawer,
   },
+  computed: {
+    langs() {
+      return ['en', 'es'];
+    },
+  }
```

Comment:
This would be dynamic based on the locales we support. We should also be adding the lang to the html of the app for accessibility.